src/main/proto/TokenCreate.proto

```diff
@@ -44,6 +44,14 @@ Note that a created token is <i>immutable</i> if the <tt>adminKey</tt> is omitte
 A token can be either <i>FUNGIBLE_COMMON</i> or <i>NON_FUNGIBLE_UNIQUE</i>, based on its <i>TokenType</i>. If it has been omitted, <i>FUNGIBLE_COMMON</i> type is used.
 
 A token can have either <i>INFINITE</i> or <i>FINITE</i> supply type, based on its <i>TokenType</i>. If it has been omitted, <i>INFINITE</i> type is used.
+
+If a <i>FUNGIBLE</i> TokenType is used, <i>initialSupply</i> should explicitly be set to a non-negative. If not, the transaction will resolve to INVALID_TOKEN_INITIAL_SUPPLY.
+
+If an <i>NON_FUNGIBLE_UNIQUE</i> TokenType is used, <i>initialSupply</i> should explicitly be set to 0. If not, the transaction will resolve to INVALID_TOKEN_INITIAL_SUPPLY.
+
+If an <i>INFINITE</i> TokenSupplyType is used, <i>maxSupply</i> should explicitly be set to 0. If it is not 0, the transaction will resolve to INVALID_TOKEN_MAX_SUPPLY.
+
+If a <i>FINITE</i> TokenSupplyType is used, <i>maxSupply</i> should be explicitly set to a positive value. If it is not, the transaction will resolve to INVALID_TOKEN_MAX_SUPPLY.
```

Comment:
Lets replace `positive value` with `non-negative`. Someone might think that for FINITE he cannot set `0`